packages/at_follows_flutter/CHANGELOG.md

```diff
@@ -1,65 +1,100 @@
+## 3.0.13:
+
+- **FIX** Depreciated AtClientService methods replaced with AtAuthService methods
```

Comment:
I think you forgot a `:` after `**FIX**`

Reply:
(You had colons on the other lines)

Reply:
Thanks, I did. Fixed